src/ORM/Marshaller.php

```diff
@@ -198,17 +199,18 @@ public function one(array $data, array $options = [])
             }
         }
 
-        if (!isset($options['fieldList'])) {
-            $entity->set($properties);
-            $entity->errors($errors);
-
-            return $entity;
+        if (isset($options['fieldList'])) {
+            $options['fields'] = $options['fieldList'];
         }
 
-        foreach ((array)$options['fieldList'] as $field) {
-            if (array_key_exists($field, $properties)) {
-                $entity->set($field, $properties[$field]);
+        if (isset($options['fields'])) {
+            foreach ((array)$options['fields'] as $field) {
+                if (array_key_exists($field, $properties)) {
+                    $entity->set($field, $properties[$field]);
```

Comment:
I'm pretty sure this gets around the potential `$_accessible` issue you @dereuromark were discussing elsewhere in this pull request.